src/main/java/sirius/db/es/LowLevelClient.java

```diff
@@ -457,6 +457,33 @@ public JSONObject closeScroll(String scrollId) {
                               .response();
     }
 
+    /**
+     * Creates a point in time (PIT) for the given alias and routing.
+     *
+     * @param alias     the alias to create the PIT for
+     * @param routing   the routing to create the PIT for
+     * @param keepAlive how long the PIT should live (initially)
```

Comment:
Add link to es time units? https://www.elastic.co/guide/en/elasticsearch/reference/current/api-conventions.html#time-units